tests/utils/test_requirements_utils.py

```diff
@@ -737,6 +744,9 @@ def predict(self, context, model_input, params=None):
 
 
 @pytest.mark.skipif(sys.version_info < (3, 10), reason="Requires Python 3.10 or higher")
+@pytest.mark.skipif(
+    not IS_DATABRICKS_AGENT_INSTALLED, reason="Requires databricks.agents to be installed"
+)
```

Comment:
can we combine `pytest.mark.skipif`?

Reply:
I guess we shouldn't combine these two as they are semantically different reasons?

Reply:
We could, but then we need to look into the codebase rather than seeing the log when we run `pytest`

```
$ pytest tests/utils/test_requirements_utils.py
->
tests/utils/test_requirements_utils.py::test_infer_pip_requirements_on_databricks_agents SKIPPED (Requires databricks.agents to be installed)

vs

tests/utils/test_requirements_utils.py::test_infer_pip_requirements_on_databricks_agents SKIPPED (See comments)
```

Reply:
How about `reason="Requires python>=3.10 and databricks-agents"`?

Reply:
Then we cannot know if the test is skipped because of the python version or databricks-agents. So I feel as long as the condition is semantically different, it's clearer to separate this type of declaration. 
Anyway, I don't have a strong preference here, changed the condition accordingly!

Reply:
@TomeHirata Thanks for the update!